### PR TITLE
Copy edits to what is quire page

### PR DESCRIPTION
**Pull request by evale124**
Addresses issue #229 

- Made slight changes to wording in certain phrases for conciseness, but kept original meaning (compare my edits to original sentence and see if you agree)
- Is it possible to limit adjectives to avoid redundancy? 
- Consider rewording second sentence of second paragraph, specifically, "long-term technology choice." Is there a clearer way of saying this without sounding wordy?
- All links work except first (Getty), but I think I addressed the problem

### CODE REVIEW

**Review by Erin-Cecele — COMMENTED** [review]
All these changes are great! Thank you Estefania! The only one that gives me pause is: 

Getty is carving a path forward for individuals and institutions looking to **solve the challenges around creating affordable, flexible, discoverable, and enduring digital publications.**
 TO 
Getty is carving a path forward for individuals and institutions looking to **solve the challenges of creating original and enduring digital publications.**

I agree that the original is too wordy. However, I feel it's important to emphasize the core concepts of Quire, the most important being **enduring and discoverable**. Maybe we trim it down to just that? 

Everything else looks great.

**Comment by geealbers** [comment]
Nice edits @evale124! And I see you made the changes @ErinCecele suggested. going to merge this in now
